### ISSUE TITLE
Xtensa support missing from Windows releases?

### ISSUE THREAD

**Issue by czietz** [radareorg/radare2#20016]
According to the [README](https://github.com/radareorg/radare2#architectures), Cadence/Tensilica Xtensa is one the supported architectures. However, if I get the [latest release for Windows](https://github.com/radareorg/radare2/releases/download/5.6.8/radare2-5.6.8-w64.zip) (5.6.8 at the time of this writing) and try to load an Xtensa binary, this merely results in:
```
asm.arch: cannot find (xtensa)
anal.arch: cannot find 'xtensa'
asm.arch: cannot find (xtensa)
anal.arch: cannot find 'xtensa'
```

Running `rasm2 -L` confirms that Xtensa support is apparently not built in.

Is this intentional? How do I get a Windows build _with_ Xtensa support? Would building it myself from source help?

**Comment by trufae** [comment]
The xtensa plugin is there, but its just not enabled in the meson builds, which is what we use to compile r2 for windows.

I can do a commit enabling this plugin on windows and you can pick the builds from the CI or compile it yourself

**Comment by czietz** [comment]
> The xtensa plugin is there, but its just not enabled in the meson builds, which is what we use to compile r2 for windows.
> I can do a commit enabling this plugin on windows and you can pick the builds from the CI or compile it yourself

Thank you for the quick reply and confirmation. If there's nothing blocking the inclusion of the xtensa support in the Windows builds, enabling it would be highly appreciated. No hurry, though, as...

> if you are in a hurry you can run sys/install.sh on any unix system and you'll get a build of r2 with that plugin compiled in

... in the meantime I'm going to use this workaround, then.

**Comment by trufae** [comment]
https://github.com/radareorg/radare2/pull/20017 waiting for the CI

**Comment by trufae** [comment]
Merged. Closing this ticket. The next 5.7 will come with xtensa architecture on the windows builds. thanks for spotting it!

**Comment by czietz** [comment]
I grabbed the _w64-meson_ build from the latest GitHub Actions run (https://github.com/radareorg/radare2/actions/runs/2315275966). While that does not complain anymore about not knowing the xtensa architecture, clearly something is still weird. The disassembly is missing:

![grafik](https://user-images.githubusercontent.com/9830685/168153442-c03b5465-e4ab-4705-b628-ded87d72e524.png)

Other architectures (e.g. Arm, x86) work.

**Comment by czietz** [comment]
@trufae : Did you have time to have a look? Disassembly is still missing for Xtensa with the current w64 build (from https://github.com/radareorg/radare2/actions/runs/2400997353).

**Comment by trufae** [comment]
Thanks for the headsup! Turns out the plugin is compiled in but it was not handling it propery internally. I just fixed that in master :) next build (and 5.7 will ship it)
